profiles/utils.py

```diff
@@ -20,6 +20,24 @@
                                           ldap_is_rtp)
 
 
+def error_handler_wrapper(func):
+    """
+    Handles bad requests that aren't caught properly. You should add this to your route function if it is routed to by
+    a variable url, i.e., '@app.route("/user/<uid>", methods=["GET"])'.
+
+    :param func: function to be wrapped
+    :return: wrapped function
+    """
+    @wraps(func)
+    def wrapped_function(*args, **kwargs):
+        try:
+            result = func(*args, **kwargs)
+            return result
+        #   pylint: disable=bare-except
+        except:
+            abort(404)
```

Comment:
I'm not sure I like returning a 404 for _any_ exception. Will this hide exceptions from sentry?

Reply:
I think it will. Caught exceptions won't bubble up

Reply:
Then @adamhb123 you should consider this requested changes, since this will kill a lot of our error tracking

Reply:
For sure I will, I've just been busy as of late. Will take a look sometime within the next few days.